src/prototype_room_attack.js

```diff
@@ -108,14 +126,17 @@ Room.prototype.attackRoom = function() {
 
   addRoom(player, this);
 
-  if (player.level === 0) {
+  if (this.controller.level < 3 && player.level === 0) {
```

Comment:
I would send them in any case. Usually a new room / player pops up and you can kill them with `attack(0)`. If they are more advanced, the can defend. Recognize that you attack a couple of time with two less and are maybe not so aware on the next `attack1`